stravalib/model.py

```diff
@@ -533,10 +793,20 @@ class ActivityLap(
 
 
 class Map(PolylineMap):
+    """Pass through object. Inherits from PolyLineMap"""
+
     pass
 
 
-class Split(Split, BackwardCompatibilityMixin, DeprecatedSerializableMixin):
+# TODO -opening an issue about perhaps a different approach to adding these attrs
+# to split()
```

Comment:
Also a TODO here I think this one has been addressed (meaning the issue is open now)

Reply:
my mistake - i missed this one. it's gone now!